src/scenes/AcceleratorRouter/ParticipantProjects/EditView/ProjectProfileEdit.tsx

```diff
@@ -0,0 +1,590 @@
+      setParticipantProjectRequestId(dispatched.requestId);
+      newInitiatedRequests.participantProject = true;
+    }
+    if (tfContact) {
+      saveTFContact();
+      newInitiatedRequests.assignTfContact = true;
+    }
+    if ((mainPhoto || mapPhoto) && photoVariableIds) {
+      const imageValues = [];
+      if (mainPhoto) {
+        imageValues.push({
+          variableId: photoVariableIds[HighlightPhotoStableId],
+          file: mainPhoto.file,
+          caption: '',
+          citation: '',
+          projectId,
+        });
+      }
+      if (mapPhoto) {
+        imageValues.push({
+          variableId: photoVariableIds[ZoneFigureStableId],
+          file: mapPhoto.file,
+          caption: '',
+          citation: '',
+          projectId,
+        });
+      }
+      const dispatched = dispatch(requestUploadManyImageValues(imageValues));
+      setUploadImagesRequestId(dispatched.requestId);
+      newInitiatedRequests.uploadImages = true;
+    }
+
+    // No requests were initiated
+    if (!Object.values(newInitiatedRequests).some(Boolean)) {
+      setRequestsInProgress(false);
+      redirectToProjectView();
+    }
+
+    setInitiatedRequests(newInitiatedRequests);
+  }, [participantProjectRecord, dispatch, projectId, mainPhoto, mapPhoto, photoVariableIds]);
+
+  const saveTFContact = useCallback(() => {
+    if (project?.organizationId && tfContact) {
+      const assignRequest = dispatch(
+        requestAssignTerraformationContact({
+          organizationId: project?.organizationId,
+          terraformationContactId: tfContact?.value,
+        })
+      );
+      setAssignTfContactRequestId(assignRequest.requestId);
+      return true;
+    }
+    return false;
+  }, [tfContact, dispatch, project?.organizationId]);
+
+  const handleOnCancel = useCallback(() => goToParticipantProject(projectId), [goToParticipantProject, projectId]);
+
+  useEffect(() => {
+    if (response?.status === 'success') {
+      setOrganizationUsers(response.data?.users);
+    }
+  }, [response]);
+
+  useEffect(() => {
+    if (participantProject) {
+      setParticipantProjectRecord(participantProject);
+    }
+  }, [participantProject, setParticipantProjectRecord]);
+
+  useEffect(() => {
+    if (!isAllowedEdit) {
+      goToParticipantProject(projectId);
+    }
+  }, [goToParticipantProject, isAllowedEdit, projectId]);
+
+  useEffect(() => {
+    if (organization?.id) {
+      const request = dispatch(requestListOrganizationUsers({ organizationId: organization.id }));
+      setOrganizationUsersRequestId(request.requestId);
+    }
+  }, [organization]);
+
+  const globalUsersWithNoOwner = useMemo(() => {
+    const ownerId = organizationUsers?.find((orgUsr) => orgUsr.role === 'Owner')?.id;
+    return globalUsersOptions?.filter((opt) => opt.value.toString() !== ownerId?.toString()) || [];
+  }, [organizationUsers, globalUsersOptions]);
+
+  return (
+    <PageWithModuleTimeline
```

Comment:
We don't need the module timeline I think, but I know this is not styled yet

Reply:
Correct ya, I'll remove it in a future story, and will be fully changing up the rest of the boxes to match design